02-Git/main.cpp

```diff
@@ -1,6 +1,38 @@
+#include <iostream>
+#include <cstdlib>
+#include <ctime>
+
+using namespace std;
+
+string weapon[3]={"rock","scissors","paper"};
+char input;
+int npc,nmy;
 
 int main(int argc, char *argv[])
-{   

```

Comment:
This line contained extra characters in first commit so this diff happen.
Please fix this in the initial commit